src/features/blog/containers/newsContainer/index.test.js

```diff
@@ -1 +1,59 @@
-xtest('test placholder', () => {})
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import FilterButton from '../../../../components/filterButton'
+import NewsCard from '../../components/newsCard'
+import NewsContainer from '.'
+
+jest.mock('../featuredArticleContainer', () => () => 'FeaturedArticleContainer')
```

Comment:
Please update the name of this component to be `FeaturedArticleContainer` so that the snapshot doesn't change.

Reply:
Fixed